openapi/listings/listings.yaml

```diff
@@ -807,21 +812,28 @@ components:
                   $ref: '#/components/schemas/timeOfDay'
               required:
                 - openDay
+                - closeDay
                 - openTime
                 - closeTime
-                - closeDay
           specialHours:
             type: array
+            description:
+              SpecialHours represents a set of time periods when a location's operational hours differ from its regular business hours. 
+              This may be a holiday or special event.
+              These hours replace the sets of hours that begin on the dates specified by the special hours entry. 
+              The `openTime` and `startDate` must predate the `closeTime` and `endDate` respectively.
+              Currently only `open` and `closed` are supported in the Vendasta platform. We intend to add others in the future.
             items:
               type: object
               properties:
                 status:
                   type: string
-                  description: |
+                  description:
                     Describes the different modes that apply to special hour periods.
-                    `specialHours` represents a set of time periods when a location's operational hours differ from its regular business hours. This may be a holiday or special event. 
-                    These hours replace all regular hours for the day. The `openTime` and `startDate` must predate the `closeTime` and `endDate` respectively.
                     Currently only `open` and `closed` are supported in the Vendasta platform. We intend to add others in the future.
+                    Overlapping `open` and `closed` periods are not allowed.
+                    When status is `open`, for the given business it will be open for the mentioned time period for the selected date. 
+                    When status is `closed`, for the given business it will be closed for the selected date.
```

Comment:
So does that mean the `openTime` and `closeTime` fields are ignored/not required when the status is `closed`?